src/Extension/CoreExtension.php

```diff
@@ -1655,4 +1655,29 @@ function twig_check_arrow_in_sandbox(Environment $env, $arrow, $thing, $type)
         throw new RuntimeError(sprintf('The callable passed to the "%s" %s must be a Closure in sandbox mode.', $thing, $type));
     }
 }
+
+    /**
+     * Check if object attribute regardless of the value, it should return true
+     * It should be able to handle when object is ActiveRecord and have null value for a column e.g $user.RoleId
+     *
+     * @param $object
+     * @param $attribute
+     * @return bool
+     */
+    function hasAttribute($object, $attribute)
+    {
+        if (!is_object($object)) {
+            return false;
+        }
+
+        try {
+            $object->$attribute;
+
+            return true;
+        } catch (Exception $exception) {
```

Comment:
Does PHP throw an exception when a property does not exist? That would be `\Exception` anyway, but I think it just emits a warning, right?

Reply:
I believe you're right, accessing random attributes didn't trigger an exception, thanks.